webui/react/src/hooks/useModal/Project/useModalProjectCreate.tsx

```diff
@@ -36,28 +36,40 @@ const useModalProjectCreate = ({ onClose, workspaceId }: Props): ModalHooks => {
       <div className={css.base}>
         <div>
           <label className={css.label} htmlFor="name">Name</label>
-          <Input id="name" maxLength={80} value={name} onChange={handleNameInput} />
+          {/* Input doesnt have value prop due to cusor jump to the end of text */}
+          <Input defaultValue="" id="name" maxLength={80} onChange={handleNameInput} />
         </div>
         <div>
           <label className={css.label} htmlFor="description">Description</label>
-          <Input id="description" value={description} onChange={handleDescriptionInput} />
+          {/* Input doesnt have value prop due to cusor jump to the end of text */}
+          <Input defaultValue="" id="description" onChange={handleDescriptionInput} />
         </div>
       </div>
     );
-  }, [ description, handleDescriptionInput, handleNameInput, name ]);
+  }, [ handleDescriptionInput, handleNameInput ]);
 
   const handleOk = useCallback(async () => {
     try {
       const response = await createProject({ description, name, workspaceId });
       routeToReactUrl(paths.projectDetails(response.id));
     } catch (e) {
-      handleError(e, {
-        level: ErrorLevel.Error,
-        publicMessage: 'Please try again later.',
-        publicSubject: 'Unable to create project.',
-        silent: false,
-        type: ErrorType.Server,
-      });
+      if (e instanceof DetError) {
+        handleError(e, {
+          level: e.level,
+          publicMessage: e.publicMessage,
+          publicSubject: 'Unable to create project.',
+          silent: false,
+          type: e.type,
+        });
+      } else {
+        handleError(e, {
+          level: ErrorLevel.Error,
+          publicMessage: 'Please try again later.',
+          publicSubject: 'Unable to create project.',
+          silent: false,
+          type: ErrorType.Server,
+        });
+      }
```

Comment:
Better error message than before
It is a bit wordy but i think the new error message is more informative

### Before

<img width="422" alt="image" src="https://user-images.githubusercontent.com/109113616/182673243-3d894398-72d2-415e-8fc1-b78c83ca3553.png">


### After

<img width="426" alt="image" src="https://user-images.githubusercontent.com/109113616/182673112-ba430a79-112b-46d2-afb0-e487750fa14f.png">